test/helper.rb

```diff
@@ -4,10 +4,14 @@
 # add the root dir to the load path
 $LOAD_PATH.unshift(File.expand_path("../..", __FILE__))
 
+TMP_PATH = File.expand_path("../../tmp", __FILE__)
+
 # require pry for debugging (`binding.pry`)
 require 'pry'
 require 'test/support/factory'
 
+# TODO - rework the tests so this isn't needed, requires removing coupling from
+# the main `Ardb` module
```

Comment:
@kellyredding - This is only needed because parts of Ardb require that the global `Ardb` is init. This is coupling which may be needed but I think it could be avoided. For example, the adapter takes an ardb config, but it shouldn't have to take the value returned from `Ardb.config`. I should be able to build a completely separate config and pass that to an adapter in the tests.

The adapter tests (like I mentioned above) and the config `RootPath` logic is the only thing I am currently aware of that is coupled to this. There may be others and it may end up being the best choice to leave it but I'd like to look into not having it if possible.